src/pages/AdminAdvancedSettings/AdminAdvancedSettings.tsx

```diff
@@ -77,25 +80,24 @@ export const AdminAdvancedSettings: React.FC = () => {
   return (
     <>
       <div className="AdminAdvancedSettings">
-        <Nav
-          className="AdminAdvancedSettings__options"
-          activeKey={selectedTab}
-          onSelect={selectTab}
-        >
+        <div className="AdminAdvancedSettings__options">
           {renderAdminAdvancedTabs}
-        </Nav>
+        </div>
       </div>
       {selectedTab === AdminAdvancedTab.basicInfo && <VenueWizard />}
       {selectedTab === AdminAdvancedTab.entranceExperience && (
         <EntranceExperience
           // @debt Venue_v2 has different structure than AnyVenue, 1 of them should be deprecated.
           venue={venue as Venue_v2}
           sovereignVenue={sovereignVenue}
-          onSave={selectDefaultTab}
+          onSave={() => history.push(adminNGSettingsUrl(venueId))}
```

Comment:
This should be a function with useCallback

Reply:
done f01366cd31785f24e9ad3565b0c8068e881e9a4e